basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
+| [Read Historical Data (tidb_snapshot)](/read-historical-data.md)                     | Y     | Y   | Y   | Y   | Y   |
+| [Optimizer Hints](/optimizer-hints.md)                                               | Y     | Y   | Y   | Y   | Y   |
+| **Data definition language (DDL)**                                                   | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Basic `CREATE`, `DROP`, `ALTER`, `RENAME`, `TRUNCATE`                                | Y     | Y   | Y   | Y   | Y   |
+| [Generated columns](/generated-columns.md)                                           | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Views](/views.md)                                                                   | Y     | Y   | Y   | Y   | N   |
+| [Sequences](/sql-statements/sql-statement-create-sequence.md)                        | Y     | Y   | N   | N   | N   |
+| [Auto increment](/auto-increment.md)                                                 | Y     | Y   | Y   | Y   | Y   |
+| [Auto random](/auto-random.md)                                                       | Y     | Y   | N   | N   | N   |
+| DDL algorithm assertions                                                             | Y     | Y   | N   | N   | N   |
+| Multi schema change: add column(s)                                                   | Y     | N   | N   | N   | N   |
+| **Transactions**                                                                     | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| Async commit                                                                         | Y     | N   | N   | N   | N   |
+| Large transactions (10GB)                                                            | Y     | Y   | N   | N   | N   |
+| [Pessimistic transactions](/pessimistic-transaction.md)                              | Y     | Y   | Y   | Y   | Experimental |
+| [Optimistic transactions](/optimistic-transaction.md)                                | Y     | Y   | Y   | Y   | Y   |
+| [Repeatable-read isolation (snapshot isolation)](/transaction-isolation-levels.md)   | Y     | Y   | Y   | Y   | Y   |
+| [Read-committed isolation](/transaction-isolation-levels.md)                         | Y     | Y   | N   | N   | N   |
+| **Partitioning**                                                                     | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [Range partitioning](/partitioned-table.md)                                          | Y     | Y   | Y   | Y   | N   |
+| [Hash partitioning](/partitioned-table.md)                                           | Y     | Y   | Y   | Y   | N   |
+| **Statistics**                                                                       | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [CMSketch](/statistics.md)                                                           | Y     | Y   | Y   | Y   | Y   |
```

Comment:
we are going to deperate the usage of CM-Sketch in TiDB 5.0. @winoros please confirm.